airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -65,6 +65,13 @@
               "description": "The name value of insight",
               "type": "string"
             },
+            "level": {
+              "title": "Level",
+              "description": "Chosen level for API",
+              "default": "ad",
+              "pattern": "^(ad|adset|campaign|account)$",
```

Comment:
Can we use ?

```json
"type" : "string",
"enum": ["ad", "adset", "campaign", "account"]
```

Reply:
on first commit 063c59056d281170183aeaa5e313fdabf8ac2b38, I tried

`
ValidLevel = Enum("ValidLevel", AdsInsights.Level.__dict__)

level: Optional[ValidLevel] = Field(
    title="Level",
    description="Chosen level for API",
    default="ad",
)
`

But integration tests didn't work with this

Reply:
@carlonuccio can you re-implement using enum? and I will  help with tests

Reply:
edited, let me know